UserService/Controllers/UserController.cs

```diff
@@ -30,4 +30,45 @@ public IActionResult CreateUser([FromBody] UserDtoCreate userCreateDto)
 
         return Created();
     }
+
+    [HttpGet]
+    public IActionResult GetAllUsers()
+    {
+        try
+        {
+            var users = _userRepository.GetAllUsers();
+
+            if (!users.Any()) return NotFound();
+
+            var usersDtos = _mapper.Map<IEnumerable<UserDtoRead>>(users);
+
+            return Ok(usersDtos);
+        }
+        catch (Exception ex)
+        {
+            return StatusCode(500, "An unexpected error occurred");
+        }
+        
+    }
+
+    [HttpGet("{id}")]
+    public IActionResult GetUser([FromRoute]string id)
+    {
+        try
+        {
+            var user = _userRepository.GetUser(id);
+
+            if (user == null)
+            {
+                return NotFound();
+            }
+
+            var userDto = _mapper.Map<UserDtoRead>(user);
+            return Ok(userDto);
+        }
+        catch (Exception ex)
```

Comment:
Maybe you should benefice from the exception throwed by using (printing) the Message from the "ex" exception pbject.

Reply:
I was thinking of leaving that until we start integrating Loki, then we can log exceptions properly.

Reply:
Sure, it's a good idea, but for debugging stuff, maybe adding the message will help us until we add Loki service